managed/src/main/java/com/yugabyte/yw/cloud/aws/AWSCloudImpl.java

```diff
@@ -14,12 +16,15 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static java.util.stream.Collectors.*;
 
 //TODO - Better handling of UnauthorizedOperation. Ideally we should trigger alert so that
 // site admin knows about it
 class AWSCloudImpl implements CloudAPI {
+  public static final Logger LOG = LoggerFactory.getLogger(AWSCloudImpl.class);
 
   // TODO use aws sdk 2.x and switch to async
   public AmazonEC2 getEcC2Client(Provider provider, Region r) {
```

Comment:
@jitendra-12113 change this to:
```
public AmazonEC2 getEcC2Client(Provider provider, Region r) {
    return getEC2ClientInternal(provider.getConfig()), r.code);
}

private getEC2ClientInternal(config , regionCode) {
    ...
}
```

Then in your code just call 
`getEC2ClientInternal`

Reply:
Done. Thanks!